filament/include/filament/Engine.h

```diff
@@ -725,6 +725,20 @@ class UTILS_PUBLIC Engine {
       */
     utils::JobSystem& getJobSystem() noexcept;
 
+    /**
+      * Tells the driver to reset any internal state tracking if necessary.
```

Comment:
You should mention this is only useful when the Filament driver and the external driver share the same thread and context (ie WebGL). This makes me wonder if the implementation shouldn't be a no-op except in WebGL? Just to avoid extra costs to apps calling this unnecessarily. @pixelflinger thoughts?